### PR TITLE
Update list.txt

### PR DESCRIPTION
**Pull request by rkmtrials**

### CODE REVIEW

**Review by aishik212 — CHANGES_REQUESTED** [review]
dont put commas in names
for example in we tv only write we tv no need to wrote moview,.....etc

**Comment by rkmtrials** [comment]
ok

**Comment by aishik212** [comment]
correct the names

File: app/src/main/res/raw/list.txt [hunk 1]
```diff
@@ -7,6 +7,29 @@ Cam Scanner,com.intsig.camscanner,com.microsoft.office.officelens,Microsoft Offi
...
+Camera360: Selfie Photo Editor with Funny Sticker,vStudio.Android.Camera360,
+MIX - Photo Editor Pro,com.pinguo.edit.sdk,
+Camera360 Lite-Selfie Camera,com.pinguo.camera360lite,
+April - Camera360 cute Layout and Template,us.pinguo.april_collage,
+Perface by Camera360,us.pinguo.perface,
+BeautyPlus - Easy Photo Editor & Selfie Camera,com.commsource.beautyplus,
+HD Camera Selfie Beauty Camera,photo.selfie.beauty.candy.camera,
+Beauty Camera Plus - Sweet Camera & Face Selfie,com.beauty.camera.youcam.makeup.photo.editor.plus,
+Mi Music,com.miui.player,
+WeTV - Dramas, Films & More,com.tencent.qqlivei18n,
```
Comment (aishik212) [inline app/src/main/res/raw/list.txt]: remove teh Drama, ....

Comment (rkmtrials) [reply app/src/main/res/raw/list.txt]: ok

File: app/src/main/res/raw/list.txt [hunk 2]
```diff
@@ -13,23 +13,23 @@ QQMusic,com.tencent.qqmusic,
...
 WPS Office Lite,cn.wps.moffice_i18n,
-WPS PDF - Free For PDF Read, View, Edit, Convert,cn.wps.pdf,
-WPS PDF Fill & Sign - Fill & Sign on PDF,cn.wps.pdf.fillsign,
+WPS PDF,cn.wps.pdf,
+WPS PDF Fill & Sign,cn.wps.pdf.fillsign,
 WPS Office Extra Goodies,cn.wps.moffice_extra,
 WPS Premium Subscription,cn.wps.moffice_premium,
 Bestie - Camera360 Beauty Cam,us.pinguo.selfie,
-Camera360: Selfie Photo Editor with Funny Sticker,vStudio.Android.Camera360,
+Camera360:,vStudio.Android.Camera360,
```
Comment (aishik212) [inline app/src/main/res/raw/list.txt:23]: remove that colon also, no need in this one

**Review by aishik212 — APPROVED** [review]

File: app/src/main/res/raw/list.txt (hunk 2 above)
Comment (rkmtrials) [reply app/src/main/res/raw/list.txt:23]: ok

**Pull request merged**